meow/user_profile/models.py

```diff
@@ -11,11 +11,12 @@ class User(AbstractUser):
         choices=ROLE_CHOICES,
         default=CONTRIBUTOR
     )
-    #where did slack_username go?
```

Comment:
we don't need the slack username since we get the meow username from slack, so they would be the same (i.e. my slack username is `dnewman` and so my meow username would be the same thing)